x-pack/plugins/ml/public/application/jobs/jobs_list/components/ml_job_editor/ml_job_editor.tsx

```diff
@@ -47,23 +48,35 @@ export const MLJobEditor: FC<MlJobEditorProps> = ({
   }
 
   return (
-    <EuiCodeEditor
+    <CodeEditor
+      languageId={mode}
+      options={{
+        readOnly,
+        theme,
+      }}
       value={value}
       width={width}
       height={height}
-      mode={mode}
-      readOnly={readOnly}
-      wrapEnabled={true}
-      showPrintMargin={false}
-      theme={theme}
-      editorProps={{ $blockScrolling: true }}
-      setOptions={{
-        useWorker: syntaxChecking,
-        tabSize: 2,
-        useSoftTabs: true,
-      }}
       onChange={onChange}
       data-test-subj={dataTestSubj}
+      editorDidMount={(editor: monaco.editor.IStandaloneCodeEditor) => {
+        const editorModelUri: string = editor.getModel()?.uri.toString()!;
+        if (schema) {
+          monaco.languages.json.jsonDefaults.setDiagnosticsOptions({
+            validate: true,
+            enableSchemaRequest: false,
+            schemaValidation: 'warning',
```

Comment:
As discussed, is it possible to show edits such as using an incorrect field type as errors rather than `warning`?

<img width="387" alt="image" src="https://user-images.githubusercontent.com/7405507/222413368-35efbedf-fa0f-4b55-bdd7-7d495fdae83f.png">

Reply:
@peteharverson you need to scroll the tooltip content all the way to the bottom 

![image](https://user-images.githubusercontent.com/5236598/222480837-bc1502b9-3418-4564-88be-bd1c6f03d6fd.png)

Reply:
The question here was whether we can show issues like this in the red error format rather than as the orange warning? What controls the category of error / warning? In this particular example, using a wrong field type for the `influencers` field for example you cannot save and close the edit flyout.

Reply:
Change in https://github.com/elastic/kibana/pull/146968/commits/95e7f19d7934d210f2e22a6bb717de627fda788b looks good, to flag these as errors rather than warning.